shared/common/ui/customScrollbar/index.tsx

```diff
@@ -17,6 +17,8 @@ export interface CustomScrollbarsProps {
   verticalBarClass?: string;
   innerClass: string | Element | null;
   headerPadding?: number;
+  scrollIgnoreLength?: number;
+  bottomScrollBarOffset?: number;
```

Comment:
Why do we need this?

Reply:
How do you mean why? :D These are the props that tells if we want to ignore some scrolling at the top, and if we want to have some offset at the bottom of the scroll bar/thumb. we need both for SideNav but none for other places where we use CustomScrollbars which is why they defaults to 0. 

Did you have other idea for this?

Reply:
Sorry, I meant just the `bottomScrollBarOffset` prop. I thought we only needed to inset the decorative track line for the docs SideNav, which doesn't need the actual scrollbar to be inset?

Reply:
I think the scroller should be inset because the thumb also should not go until the screen edge or below.

Reply:
if we inset only the decorative line (scroll bar) the thumb will continue scroll after the line finish which is weird, we want to finish scrolling before the screen edge